apollo/smoothie.py

```diff
@@ -3,52 +3,18 @@
 import json
 import concurrent
 
+logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger()
 
-# smoothie = SmoothieTalker()
-# smoothie.connect()
-#
-# smoothie.send("gcode")
-# res = smoothie.get()
-#
-#
-# q = []
-#
-#
-# while True:
-#     res = yield smoothie.read()
-#     print(res)
-#
-#     q.append(input())
-#
-#     while q:
-#         if smoothie.is_ready():
-#             yield smoothie.send(q.pop(0))
-#             print(yield from smoothie.read())
-#         else:
-#             cnt += 1
-#             sleep(1)
-#
-#
-# class SmoothieTestCase(...)
-#     def test_send(self):
-#         smoothie.transport.write = Mock(return_value={stat: ok})
-#         smoothie.send()
-#
-#         assert smoothie.transport.write.called == True
-#
-#     def read(self):
-#         pass
-#
-
-
-
-class SmoothieBoard(object):
-    def __init__(self, host, port, loop, feedback=False):
+class SmoothieMaxReadsException(Exception):
+    def __init__(self, gcode=None):
```

Comment:
With exceptions, do you not need to super() (aka the parent) init? And does that harm anything if not...